06. Errors&Exceptions/03. OwnContextManager.py

```diff
@@ -0,0 +1,24 @@
+# 3. Write your own context manager whose task will be to print "==========" - 10 characters are equal before code
+# execution and after code execution, thus highlighting the block of code with equals characters.
+
+class OwnContextManager:
+    def __init__(self, chars: str = "=========="):
+        self.chars = chars
+
+    def __enter__(self):
+        print(self.chars)
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        print(self.chars)
+
+
+def power(a, b):
+    print(a**b)
+
+
+with OwnContextManager() as value:
```

Comment:
You don't return any value from the context manger and can skip part `as value` here since you don't use it